ops/scripts/pre-deployment-check.sh

```
@@ -75,146 +77,306 @@ redis_persistence_ok() {
+    pass_check "node_modules present"
+  fi
+else
+  pass_check "node_modules present"
+fi
+
+# ============================================================================
+# CHECK 11: Orphaned build processes (past crashes leave zombies that eat RAM)
+# ============================================================================
+CHECK=$((CHECK + 1))
+echo "[$CHECK/$TOTAL_CHECKS] Orphaned build processes..."
+ORPHANS=$(ps aux | grep -E "turbo.*build|next build" | grep -v grep | grep -v "$$" || true)
```

Shellcheck (0.11.0)

[info] Consider using pgrep instead of grepping ps output.

(SC2009)